finagle-zipkin/src/main/scala/com/twitter/finagle/zipkin/thrift/ScribeRawZipkinTracer.scala

```diff
@@ -0,0 +1,184 @@
+package com.twitter.finagle.zipkin.thrift
+
+import com.google.common.io.BaseEncoding
+import com.twitter.conversions.storage._
+import com.twitter.conversions.time._
+import com.twitter.finagle.builder.ClientBuilder
+import com.twitter.finagle.stats.{ClientStatsReceiver, NullStatsReceiver, StatsReceiver}
+import com.twitter.finagle.thrift.{Protocols, ThriftClientFramedCodec}
+import com.twitter.finagle.tracing._
+import com.twitter.finagle.util.DefaultTimer
+import com.twitter.finagle.zipkin.core.{TracerCache, Span, RawZipkinTracer}
+import com.twitter.finagle.zipkin.thriftscala.{ResultCode, LogEntry, Scribe}
+import com.twitter.finagle.{Service, SimpleFilter}
+import com.twitter.scrooge.TReusableMemoryTransport
+import com.twitter.util._
+import java.io.CharArrayWriter
+import java.net.InetSocketAddress
+import java.util.concurrent.ArrayBlockingQueue
+import org.apache.thrift.TByteArrayOutputStream
+import scala.collection.mutable.ArrayBuffer
+import scala.language.reflectiveCalls
+
+object ScribeRawZipkinTracer {
```

Comment:
Yes it should be unaffected.
`META-INF/services/com.twitter.finagle.tracing.Tracer` in finagle-zipkin points to [`com.twitter.finagle.zipkin.thrift.SamplingTracer`](https://github.com/twitter/finagle/pull/513/files#diff-06cb96e4d4229bad23c2e59316178504R61).